next/components/thumbprint-components/props-table/component-definition.ts

```diff
@@ -0,0 +1,24 @@
+export interface ComponentDefinition {
+    displayName: string;
+    description: string;
+    methods: unknown[];
+    props: Record<
+        string,
+        {
```

Comment:
Can this be a named type to make the nesting more readable?

Reply:
Done